vaadin-combo-box.html

```diff
@@ -251,7 +251,10 @@
         } else if (this._isEventKey(e, 'esc')) {
           this._onEscape();
         }
-      }.bind(this));
+      }.bind(this);
+
+      this.$.input.addEventListener('keydown', keydownHandler);
+      this.$.fullScreenInput.addEventListener('keydown', keydownHandler);
```

Comment:
Would the keydown event happen to bubble up to the root element, so we could just use one event listener?